migrations/sql/V2020.11.05.13.09__create_now_delay.sql

```diff
@@ -0,0 +1,31 @@
+
+    CREATE TABLE IF NOT EXISTS now_application_delay_type ( 
+        delay_type_code varchar PRIMARY KEY,
+        description varchar,
+        display_order integer,
+        active_ind boolean NOT NULL default 'True',
+        create_user character varying NOT NULL,
+        create_timestamp timestamp with time zone DEFAULT now() NOT NULL,
+        update_user character varying NOT NULL,
+        update_timestamp timestamp with time zone DEFAULT now() NOT NULL
+    );
+
+    CREATE TABLE IF NOT EXISTS now_application_delay (
+        now_application_delay_id SERIAL primary key, 
+        now_application_delay_guid   uuid DEFAULT gen_random_uuid()   UNIQUE NOT NULL ,
+        now_application_guid uuid, 
```

Comment:
`now_application_guid uuid NOT NULL`?
`delay_type_code varchar NOT NULL`?

Reply:
Good eyes, fixed.